ui/app/styles/_layout.scss

```diff
@@ -128,16 +128,6 @@ hr {
   border-top: 1px solid rgb(var(--border));
 }
 
-// Set a default size for icons
-
-svg.icon {
-  width: scale.$lg-2;
-  height: scale.$lg-2;
-  flex-shrink: 0;
-}
-
-.pds-icon {
-  display: inline-flex;
-  align-items: center;
-  justify-content: center;
+button .icon {
```

Comment:
should this style be moved to button.scss? 

also this is just for the `button` element, not the button css classes, so:
- do button-style links with icons still render correctly?
- do `button` elements with icons, used without the button style (for accessibility) still render properly?

Reply:
It looks like this gets overridden by more-specific rules in `button.scss`:

<img width="483" alt="CleanShot 2021-12-14 at 16 37 31@2x" src="https://user-images.githubusercontent.com/34030/146029734-248893a4-6f1f-4c22-957e-e81bc21cf780.png">

Maybe we don’t need it at all?